### ISSUE TITLE
Property value? Code is broken after updating.

### ISSUE THREAD

**Issue by shustrya** [lovata/oc-shopaholic-plugin#87]
Test code:
```
[ProductList]
==

{% set obProductList = ProductList.make().active().random(2) %}
{# % set obProductList =  ProductList.make().tag(3) % #}

<div class="uk-margin-top" uk-slider="autoplay: true; autoplay-interval: 5000;">
<div class="uk-slider-container uk-light">
<ul class="uk-slider-items uk-child-width-1-2@s uk-child-width-1-4@m  uk-grid">

{% if obProductList.isNotEmpty() or obProductList is not empty %}

 {% for obProduct in obProductList %}
            {% set obOffer = obProduct.offer.active().first() %}
            {% set obGroup = obProduct.property.getGroupList().getByCode('shop_images') %}
            <li>
            <div class="uk-text-center tsn-color-white">
                <div class="uk-card uk-card-default">
                    <div class="uk-card-media-top uk-inline">

                      {# Get property collection, apply sorting and filters by active, group #}
                      {% set obPropertyList = obProduct.property.group(obGroup.id) %}
                      {% if obPropertyList.isNotEmpty() %}
                         {% for obProperty in obPropertyList if obProperty.hasValue() %}
...
   <!-- WRONG!!!  -->    <img src="{{ obProperty.property_value.getValueString() |media }}" alt="" width="100%" height="auto">
...
                         {% endfor %}

                      {% else %}
                          <img src="{{ '/static/noimage43.png' |media }}" alt="" width="100%" height="auto">
                      {% endif %}

                    </div>

                </div>
            </div>
          </li>
 {% endfor %}

{% endif %}

</ul>
</div>
</div>
```

This code generate wrong(same) image path. 
The property values are not the same and have the correct values.
For the first item: 0425_krokodile-img-1.jpg
For the second item: 0543_vetka-img-1.jpg

But we have this( src="..."):
```
...
  <a class="tsn-color-black" href="/shop/product/0425_krokodile">
<img src="/storage/app/media/shop/0425_krokodile/0425_krokodile-img-1.jpg" alt="" width="100%" height="auto">
</a>

  <a class="tsn-color-black" href="/shop/product/0543_vetka">
<img src="/storage/app/media/shop/0425_krokodile/0425_krokodile-img-1.jpg" alt="" width="100%" height="auto">
</a>
...
```

**Comment by kharanenka** [comment]
Hi! Do you have Toolbox plugin with version 1.10.1?

**Comment by shustrya** [comment]
Hi! Yes, I have version 1.10.1 installed.
In addition, I was trying to reinstall all the plugins. Without a result.

**Comment by kharanenka** [comment]
Fixed in Toolbox plugin version 1.10.2
